source/_integrations/backup.markdown

```diff
@@ -21,9 +21,12 @@ related:
     title: Home Assistant Operating System- Restoring a backup
 ---
 
-The **Backup** {% term integration %} is used for {% term "Home Assistant Core" %} and {% term "Home Assistant Container" %} installations to create and download backups. This backup file can be used if you migrate to {% term "Home Assistant Operating System" %}.
+The **Backup** {% term integration %} is used for {% term "Home Assistant Core" %} and {% term "Home Assistant Container" %} installations to create and download backups, which are stored in a subdirectory `backups`, located in your [configuration directory](/docs/configuration/#editing-configurationyaml).
```

Comment:
I don't think this is fitting for the lead paragraph. Let's add some additional context in a separate paragraph.

Reply:
Yes, you're right. That ideed makes it even better 👍